server/src/database/queries/search.ts

```
@@ -0,0 +1,1198 @@
+      switch (field) {
+        case 'name':
+          fieldScore = this.calculateTextSimilarity(reference.name, candidate.name)
+          break
+        case 'content':
+          fieldScore = this.calculateTextSimilarity(reference.content, candidate.content)
+          break
+        case 'type':
+          fieldScore = reference.node_type === candidate.node_type ? 1.0 : 0.0
+          break
+        case 'tags':
+          const refTags = this.extractTags(reference.content)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)